### PR TITLE
Update colabs to be tf2 compatible

### PR DESCRIPTION
**Pull request by jameswex**
Updated all notebooks to work in tf2. All notebooks except text toxicity work in both tf1 and tf2.
Text toxicity only works in tf2, but includes ensuring tf2 is installed in its first cell. This is because its use of tf hub and keras models could not be done in a way to be compatible in both, and we want to show tf2 code moving forward.

### CODE REVIEW

File: WIT_Toxicity_Text_Model_Comparison.ipynb [hunk 1]
```diff
@@ -49,7 +51,7 @@
         "# If running in colab then pip install, otherwise no need.\n",
         "try:\n",
         "  import google.colab\n",
-        "  !pip install --upgrade witwidget\n",
+        "  !pip install --upgrade tensorflow>=2.0.0 witwidget \n",
```
Comment (tolga-b) [inline WIT_Toxicity_Text_Model_Comparison.ipynb]: do we still need this or does colab come with tf pre-installed?

File: WIT_Toxicity_Text_Model_Comparison.ipynb [hunk 2]
```diff
@@ -115,12 +117,12 @@
...
         "id": "nStoYhqT80WH",
         "colab_type": "code",
-        "cellView": "form",
+        "cellView": "both",
         "colab": {}
       },
       "source": [
         "#@title Define custom prediction functions so that WIT infers using keras models\n",
-        "from keras.preprocessing.sequence import pad_sequences\n",
+        "import tensorflow as tf\n",
```
Comment (tolga-b) [inline WIT_Toxicity_Text_Model_Comparison.ipynb]: importing tensorflow second time?

File: WIT_Toxicity_Text_Model_Comparison.ipynb (hunk 1 above)
Comment (jameswex) [reply WIT_Toxicity_Text_Model_Comparison.ipynb]: this ensures that even if someone runs this colab on a tf1 kernel, it installs tf2 and so doesn't fail. with a tf2 kernel this is a no-op

Comment (jameswex) [reply WIT_Toxicity_Text_Model_Comparison.ipynb]: just ensures that we use tf2 even in tf1 colab kernel, as this notebook relies on tf2

Comment (jameswex) [reply WIT_Toxicity_Text_Model_Comparison.ipynb]: ensures tf2 is used even in tf1 kernels

**Pull request merged**